applications_tests/gls_sharp_navier_stokes_2d/moving_rectangle.prm

```diff
@@ -0,0 +1,124 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+subsection simulation control
+  set method                  = bdf1
+  set time step               = 0.03125
+  set number mesh adapt       = 3
+  set time end                = 0.125
+  set subdivision             = 1
+end
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+  subsection fluid 0
+    set kinematic viscosity            = 1
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+    set type                 = dealii
+    set grid type            = subdivided_hyper_rectangle
+    set grid arguments       = 1,1:-2,-2 : 2,2 : true
+    set initial refinement   = 4
+end
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+subsection FEM
+    set velocity order        = 1
+    set pressure order        = 1
+end
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+    set number                  = 0
+end
+
+# --------------------------------------------------
+# IB particles
+#---------------------------------------------------
+subsection particles
```

Comment:
Good catch! Yes I reformatted the `=`
This subsection is in the Resolved CFD-DEM parameters guide

Reply:
Oh ok! Maybe it could be good one day to make sub-pages for each `subsection` of this solver also (like for CFD), so that it will pop more easily for a search on the documentation. Food for thoughts :)